### ISSUE TITLE
Portable GUI

### ISSUE THREAD

**Issue by vbjay** [gitextensions/gitextensions#5780]
**Do you want to request a *feature* or report a *bug*?**
Feature
**What is the current behavior?**
There is no way for the user to know if the app is running as a portable configuration.  A portable configuration is where the gitextensions.exe.config file has IsPortable set to true.  This controls where settings are stored and such.  
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**
A user notable UI that tells the user they are running under a portable config and not storing settings in user profile.
**Environment you encounter the issue:**

- Git Extensions 3.00.rc2
- c68db7bc3fe70a159ed409048f27d1f39c25d601 
- Git 2.19.1.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3190.0
- DPI X:100.00% Y:100.00%

**Did this work in previous version of GitExtensions (which)?**

**Comment by RussKie** [comment]
Add `(portable)` to the title bar?

**Comment by gerhardol** [comment]
Suggest info in about form

**Comment by vbjay** [comment]
Ok. I think both are good. I want a more noticeable UI so title and about.
 About can have a detailed info. Translate "Portable" in titlebar?

**Comment by RussKie** [comment]
> Translate "Portable" in titlebar

Yes, if we choose to use it

> Will normal users constantly need to be reminded what variant they are using?

I think it is wise to show the user this info.
I am not sufficiently familiar with "portable" behaviour of GE, but I presume it stores all settings locally. Is this correct, @vbjay?

**Comment by vbjay** [comment]
Yes.  @jbialobr and I got it going a while ago.  The goal was to make it so you could have it on a thumb drive or what have you and keep your settings with you.  It's not as portable as I would like but I was trying to make it so I could give students a zip to download and they could use it in the computer lab and other school computers.  When IsPortable is true, settings are stored in the program folder.  

Still lacking are putty, git...finding in a relative folder so you don't have different drive letters messing things up.

**Comment by vbjay** [comment]
See #2556
